patches/chrome-browser-resources-settings-route.js.patch

```diff
@@ -1,16 +1,21 @@
 diff --git a/chrome/browser/resources/settings/route.js b/chrome/browser/resources/settings/route.js
-index 866453bdfddf5c899472633b11910ad41de81e35..72aad165de9c687110e3696e3bdbf8c58b8580eb 100644
+index 866453bdfddf5c899472633b11910ad41de81e35..eb0a03558de4637cd7df733fc2795023db8d20ed 100644
 --- a/chrome/browser/resources/settings/route.js
 +++ b/chrome/browser/resources/settings/route.js
-@@ -331,6 +331,7 @@ cr.define('settings', function() {
+@@ -328,9 +328,12 @@ cr.define('settings', function() {
+ 
+       // TODO(tommycli): Find a way to refactor these repetitive category
+       // routes.
++<if expr="_google_chrome">
```

Comment:
Shouldn't this have a comment indicator before it? The other things in this file do.